api-tests/dev_apis/crypto/test_c077/test_c077.c

```diff
@@ -472,6 +472,12 @@ int32_t psa_pake_get_shared_key_test(caller_security_t caller __UNUSED)
              TEST_ASSERT_DUAL(status, check1[i].expected_status[0],
                                       check1[i].expected_status[1], TEST_CHECKPOINT_NUM(12));
 
+             if (status == PSA_SUCCESS) {
```

Comment:
If the shared_key is destroyed by calling psa_destroy_key() here, same key identifier can not be used to extract shared key from server. To avoid this we can have two key identifiers for client and server as shared_ckey and shared_skey, shared_key as key identifier for JPAKE user shared secret.

Reply:
I think this is a misunderstanding of the specification text - which is understandable, given the current wording in `psa_destroy_key()`:

> Destroying the key makes the key identifier invalid, and the key identifier must not be used again by the application.

This is meant to indicate that the key identifier cannot be used [to refer to the key that has been destroyed]. Using that key identifier for an operation will result in a `PSA_ERROR_INVALID_HANDLE` error, _until_ a new key is created with the same key identifier value.

But note:

1. It is not good practice for an application to create persistent keys for different purposes using the same identifier, even if they never exist at the same time. This could lead to key usage errors that can be avoided by using distinct key identifier values.
2. Volatile key identifier values are allocated by the implementation - a robust implementation should not immediately reuse volatile key identifier values, to increase the likelihood that an application use-key-after-destroy error is detected.